apps/server/src/modules/room/domain/do/room-authorizable.ts

```diff
@@ -0,0 +1,8 @@
+import { AuthorizableObject, DomainObject } from '@shared/domain/domain-object';
+import { RoomMember } from '@src/modules/room-member';
+
+export interface RoomAuthorizableProps extends AuthorizableObject {
```

Comment:
this should probably be part of the RoomMembers module, since for authorization we pretty much only need the members. Maybe we dont even need a seperate class, but can just use the RoomMembers DO to authorize rooms

Reply:
Yes moved to RoomMember